xbmc/windowing/X11/WinSystemX11.cpp

```diff
@@ -764,7 +764,7 @@ bool CWinSystemX11::SetWindow(int width, int height, bool fullscreen, const std:
       Atom fs = XInternAtom(m_dpy, "_NET_WM_STATE_FULLSCREEN", True);
       XChangeProperty(m_dpy, m_mainWindow, XInternAtom(m_dpy, "_NET_WM_STATE", True), XA_ATOM, 32, PropModeReplace, (unsigned char *) &fs, 1);
       // disable desktop compositing for KDE, when Kodi is in full-screen mode
-      int one = 1;
+      long one = 1;
```

Comment:
can this be a sized integer instead?

Reply:
With sized integer do you mean `int32_t`/`int64_t`? In that case no as the spec requires `long` and `long`s size is platform/architecture dependent.

Reply:
No, the proper type should be "long". This means it will be processed as 64 bits on x86_64 and 32 bits on arm32.